src/main/resources/static/styles.css

```diff
@@ -32,4 +32,5 @@ textarea {
     margin-bottom: 15px;
     border: 1px solid #ccc;
     border-radius: 5px;
```

Comment:
# Change 3

This change isn't really shown as easily in the git diff, but I moved this file to a `/static` folder. This css file wasn't able to be imported by the pages before this change.

https://www.baeldung.com/spring-mvc-static-resources